### DISCUSSION TITLE
v1.0.0-rc5

### DISCUSSION THREAD

**Discussion by Mrs4s** [Mrs4s/go-cqhttp#1992] [General]
## Changelog
本次版本修复了包括 `账号密码错误` 等许多问题, 强烈建议升级.
在最近的更新中，QQ修复了扫码登录手机等协议的漏洞，今后只有`Android Watch`协议支持扫码登录。
从这个版本开始, 默认情况下需要访问 `captcha.go-cqhttp.org` 域名来获取签名, 请确保该域名未被DNS污染.
本次更新后请务必删除 `session.token` 和 `device.json` 文件重新创建.

### 新增
86f5b7f 支持了私聊文件发送. @wdvxdr1123 
2a4ea28 支持自动转换 `webp` 格式的图片以解决 `Windows QQ` 无法正常显示的问题. @MaikoTan 
fc0845b 新增 `set_group_anonymous` API. @BeautyyuYanli 
17420fe 为 `get_stranger_info` API 添加了 `sign` 字段. @wdvxdr1123 
0312f05 支持了部分协议的自动更新, 今后协议将不再跟随版本更新. @Mrs4s 
c3840a5 优化了 `Windows` 系统下的 `cmd` 体验. @fumiama 
a0e3291 为登录错误显示了具体的错误代码. @Mrs4s 

### 修复
7d5f1d6 修复单向好友的消息无法发送的问题. @Mrs4s 
4d064e1 修复通过 `download_file` API下载的文件没有正确释放的问题. @Mrs4s 
d96f840 修复取出消息时`LocalImageElement`缺失的问题 @ctaoist 
2483eb0 修复当 `set_group_ban` 的 `duration` 正好等于 `2592000` 时没有正确判断报错的问题. @Pandaft 
a704009 修复部分登录错误会被误判为 `密码错误或账号被冻结` 的问题. @Mrs4s 
82ecf19 修复一处可能导致`[CQ:at]`消息段解析错误的问题. @Mrs4s 

### 协议修复
修复了由于各种原因导致的密码登录失效的问题.
完善了登录协议的实现

### 其他
标准库更新为 `go1.20`
为 `onebot v12` 提供了初步的API支持

<hr /><em>This discussion was created from the release <a href='https://github.com/Mrs4s/go-cqhttp/releases/tag/v1.0.0-rc5'>v1.0.0-rc5</a>.</em>

**Comment by wei-z-git** [comment]
热烈庆祝v1.0.0-rc5发布，感谢楼主！ /庆祝

**Comment by lmst2** [comment]
太好了，终于可以用密码登录了

**Comment by 120L020830** [comment]
请问使用云服务器出现以下情况怎么解决
![image](https://user-images.githubusercontent.com/104428954/227868609-e4eeb2e8-19f1-46ca-8148-4ce8cdb18994.png)

**Reply by wei-z-git** [reply]
用协议2+扫码比较稳定

**Reply by 120L020830** [reply]
确实，改成2就可以了

**Comment by anotherATTCK** [comment]
似乎又不能用了
![image](https://user-images.githubusercontent.com/107333389/227947655-196f7305-2221-49ff-ad15-1ce6d1d61f2b.png)

**Reply by Rxh77777** [reply]
![image](https://user-images.githubusercontent.com/72742755/228115769-cb487582-0761-4cda-a883-e75a8d3dd0a7.png)

**Comment by Rxh77777** [comment]
长消息发送失败[WARNING]: 群消息发送失败: 账号可能被风控.

**Reply by zwa73** [reply]
一样的问题，但是让机器人发送一下短消息就又好了

**Comment by blzckcxt** [comment]
尝试一下

**Comment by Rxh77777** [comment]
获取不到这一类的信息，'post_type': 'notice', 'notice_type': 'group_recall'
